package.json

```diff
@@ -128,19 +128,18 @@
     "pretest": "pnpm run compile-tests && pnpm run compile && pnpm run lint",
     "lint": "eslint src --ext ts",
     "test": "node ./out/test/integration/runTest.js",
-    "unit:test": "jest",
-    "unit:test-watch": "jest --watch"
+    "unit:test": "mocha -r ts-node/register 'src/test/unit/**/*.spec.ts'"
```

Comment:
can you please add this step to the CI? ( github action )

Reply:
I already did that here
d28c1f757296675b90aa7ecfdcc49bc48f835175